.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -11,4 +11,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: gradle/wrapper-validation-action@v3
+      - uses: gradle/actions/wrapper-validation@v3
```

Comment:
should we upgrade to the current version (v4)?

Reply:
I think the current version is still v3
https://github.com/gradle/wrapper-validation-action

Reply:
I was referring to the non-deprecated replacement:
gradle/actions/wrapper-validation@v4

see https://github.com/gradle/actions/tree/main/wrapper-validation#example-workflow

Reply:
OK, got it.